apps/client/src/features/app-settings/panel/project-panel/ProjectPanel.tsx

```diff
@@ -1,20 +1,78 @@
+import { useState } from 'react';
 import { Button } from '@chakra-ui/react';
+import { AxiosError } from 'axios';
 
+import { PROJECT_LIST } from '../../../../common/api/apiConstants';
+import { createProject } from '../../../../common/api/ontimeApi';
+import { ontimeQueryClient } from '../../../../common/queryClient';
 import * as Panel from '../PanelUtils';
 
+import ProjectForm, { ProjectFormValues } from './ProjectForm';
 import ProjectList from './ProjectList';
 
+import style from './ProjectPanel.module.scss';
+
 export default function ProjectPanel() {
+  const [isCreatingProject, setIsCreatingProject] = useState(false);
+  const [submitError, setSubmitError] = useState<string | null>(null);
+
+  const handleToggleCreate = () => {
+    setIsCreatingProject((prev) => !prev);
+    setSubmitError(null);
+  };
+
+  const handleSubmitCreate = async (values: ProjectFormValues) => {
+    try {
+      setSubmitError(null);
+
+      if (!values.filename) {
+        setSubmitError('Filename cannot be blank');
+        return;
+      }
+      await createProject(values.filename);
```

Comment:
question: what is the expected behaviour for a user that creates a new project? Should the new project be loaded after the creation is successful?

Reply:
I thought it only created it, and left if there on the list

isn't creating + loading a part of quick start?

Reply:
yes, lets talk about this again when we work on quick start